website/next.config.js

```diff
@@ -23,56 +21,85 @@ const env = {
 }
 
 const withNextra = nextra({
-  theme: './src/nextra-theme/index.tsx',
+  theme: './src/NextraLayout.tsx',
   search: false,
   codeHighlight: false,
   defaultShowCopyCode: false,
   readingTime: true,
   transformPageMap(pageMap) {
-    const locale = pageMap[0].data.slice(0, 2)
+    const locale = pageMap[0].route.slice(1, 3)
```

Comment:
Hey @dimaMachina I'm getting this error after checking out this branch and running `pnpm install` and `pnpm build`:

> TypeError: Cannot read properties of undefined (reading 'route')

If I `console.log({ pageMap })` before this line, I get `{ pageMap: [] }`. What am I doing wrong?